src/common/dao/project_blob.go

```diff
@@ -35,22 +35,43 @@ func AddBlobToProject(blobID, projectID int64) (int64, error) {
 
 // AddBlobsToProject ...
 func AddBlobsToProject(projectID int64, blobs ...*models.Blob) (int64, error) {
+	blobCounts := len(blobs)
 	if len(blobs) == 0 {
 		return 0, nil
 	}
 
-	now := time.Now()
+	// 65535 is the maximum support parameters, if larger than 65535, will divide them into chunks with size 10000
+	var dividedBlobs [][]*models.Blob
```

Comment:
I think even when the length of the `blobs` array is less then 65535, we should divide them into chunks, maybe 1024 or 4096 records in one chunk.

And this is a little trivial but seems you can do that in one loop.

Reply:
will update it to 1024

Reply:
use the single insert instead